doc/man7/openssl-quic.pod

```diff
@@ -498,6 +504,57 @@ The following SSL APIs are new but relevant to both QUIC and DTLS:
 
 =over 4
 
+=item L<SSL_new_listener(3)>
```

Comment:
This function and the others below are in the wrong list. This list is preceded by the text "The following SSL APIs are new but relevant to both QUIC and DTLS". But all of these currently only work for QUIC not DTLS (feasibly we could extend them for DTLS in the future). There is a later list with the text "The following SSL APIs are specific to QUIC". This documentation should be moved there instead.